app/views/user_mailer/change_email.html.erb

```diff
@@ -1,3 +1,3 @@
 <% content_for :message do %>
-  <%= t('.part1', login: style_bold(@user.login), email: style_email(@user.email)).html_safe %>
+  <%= t('.changed', login: style_bold(@user.login), email: style_email(@user.email)).html_safe %>
```

Comment:
Do we want to do anything about this `html_safe` while we're here? I really can't decide if we should let it go for now or not.

Reply:
Let's fix that here, we did talk about [using the `html` suffix in keys](https://otwarchive.atlassian.net/browse/AO3-6240?focusedCommentId=359954).